### PR TITLE
Background improvements

### LINKED ISSUE #82: Add indication for background image size and resolution
- [x] Remove scale parameter.

### LINKED ISSUE #83: Change background default color

### PR DESCRIPTION
**Pull request by swouf**
Close #82 
Close #83

### CODE REVIEW

File: src/components/modes/teacher/BackgroundToggle.tsx [hunk 1]
```diff
@@ -84,91 +72,25 @@ const BackgroundToggle = (): JSX.Element => {
...
-          }}
-          onChange={handleScaleChange}
-        />
-      </ToggleContainer>
-    </>
+    <ToggleContainer>
+      <Typography fontSize="1.1em" color={toggleDisabled ? grey[500] : 'black'}>
+        {t('Show Background Image')}
+      </Typography>
+      <FormControlLabel
```
Comment (spaenleh) [inline src/components/modes/teacher/BackgroundToggle.tsx:80]: Is there a reason why you use the `FormControlLabel` with a label of undefined ? Would `FormControl` be more suitable for your use-case ?

File: src/components/modes/teacher/ImageUpload.tsx [hunk 2]
```diff
@@ -120,6 +124,12 @@ const ImageUpload = (): JSX.Element => {
         <Typography sx={{ color: '#383838' }}>
           {t(`Max file size: ${MAX_FILE_SIZE / 1e6}mb`)}
         </Typography>
+        <Typography sx={{ color: '#383838' }}>
+          {t(
+            'Recommended image size for full background: {{ CANVAS_WIDTH_PX }}x{{ CANVAS_HEIGHT_PX }} px.',
```
Comment (spaenleh) [inline src/components/modes/teacher/ImageUpload.tsx:129]: I think you need to add it to your translations 🎈

File: src/components/modes/teacher/ImageUpload.tsx [hunk 3]
```diff
@@ -120,6 +124,12 @@ const ImageUpload = (): JSX.Element => {
         <Typography sx={{ color: '#383838' }}>
           {t(`Max file size: ${MAX_FILE_SIZE / 1e6}mb`)}
         </Typography>
+        <Typography sx={{ color: '#383838' }}>
```
Comment (spaenleh) [inline src/components/modes/teacher/ImageUpload.tsx:127]: You can factor your typography out into a custom styled component:
```ts
const StyledText = styled(Typography)({
  color: '#383838',
});
``` 
And then use it here and above.

**Review by spaenleh — APPROVED** [review]
Looks good, so you dropped the scaled background feature I see 🖼️ 
I made some comments and suggestions, feel free to ask for another review or merge if they don't apply

File: src/components/modes/teacher/BackgroundToggle.tsx (hunk 1 above)
Comment (swouf) [reply src/components/modes/teacher/BackgroundToggle.tsx:80]: I think this is partially updated legacy code. I completely removed the form control label.

File: src/components/modes/teacher/ImageUpload.tsx [hunk 4]
```diff
@@ -32,6 +32,8 @@ const ImageUploadContainer = styled('div')(() => ({
   marginBottom: 1,
 }));
 
+const SubTypography = styled(Typography)(() => ({ color: '#383838' }));
```
Comment (spaenleh) [inline src/components/modes/teacher/ImageUpload.tsx:35]: Even this should work, but no worries.
```suggestion
const SubTypography = styled(Typography)({ color: '#383838' });
```

**Review by spaenleh — APPROVED** [review]
Looks good ! It is ready to merge !

**Pull request merged**